tests/Basics/icu-helper.cpp

```diff
@@ -56,7 +56,9 @@ void IcuInitializer::setup(char const* path) {
                                                      p, "basics_suite");
   if (icuDataPtr == nullptr ||
       !arangodb::basics::Utf8Helper::DefaultUtf8Helper.setCollatorLanguage(
-          "", icuDataPtr)) {
+          "", icuDataPtr, true) ||
```

Comment:
This code by || condition will always try to set both collator languages (default and ICU). What was the intention behind this code ?